server/src/data/bookings.ts

```diff
@@ -0,0 +1,17 @@
+import * as firestore from "firebase/firestore";
+const db = require("../firebase/config").db;
+const doc = firestore.doc;
+
+export const cancelBooking = async (bookingId: string) => {
+  await firestore.updateDoc(doc(db, "bookings", bookingId), {
+    status: "CANCELLED",
```

Comment:
Great job. Im just thinking maybe if its cancelled we should just delete it because we wouldn't need it in our database anymore right?

Reply:
If we didn't need it anymore then yeah we could delete it. But I think our plan is to display a user's previous and upcoming bookings, so that's why we just change the status so it can still appear.

Reply:
yeah simple alteration if we need to change later on but looks good overall

Reply:
yeah this would be for the user history or both booker and listing owner